app/views/votes/_voting.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if Vote.exists?(quote: quote, user: current_user) %>
+  <%= link_to "I hate it", quote_vote_path(quote, vote, { vote: { value: -2 } } ), method: :patch %>
```

Comment:
Magic numbers like -2 and -1 don't really mean anything. We should have this be enum'd in some fashion.